stories/__tests__/SQFormDatePicker.stories.test.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import {LocalizationProvider} from '@material-ui/pickers';
+import MomentAdapter from '@material-ui/pickers/adapter/moment';
+import {composeStories} from '@storybook/testing-react';
+import {render, screen, within} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import * as stories from '../SQFormDatePicker.stories';
+
+const {BasicDatePicker} = composeStories(stories);
+
+const renderDatePicker = props => {
+  render(
+    <LocalizationProvider dateAdapter={MomentAdapter} locale={'en'}>
+      <BasicDatePicker {...props} />
+    </LocalizationProvider>
+  );
+};
+
+describe('SQFormDatePicker Tests', () => {
+  it('should render a label and input', () => {
+    renderDatePicker();
+
+    const label = screen.getByText('Date');
+    expect(label).toBeInTheDocument();
+    expect(label).toHaveTextContent('Date');
+
+    const textField = screen.getByRole('textbox', {name: /date/i});
+    expect(textField).toBeInTheDocument();
+    expect(textField).toHaveValue('09/22/2020');
+  });
+
+  it('should render default with empty initial value', () => {
+    const initialValues = {
+      date: ''
+    };
+
+    renderDatePicker({SQFormProps: {initialValues}});
+
+    const textField = screen.getByRole('textbox', {name: /date/i});
+    expect(textField).toHaveValue('');
+  });
+
+  it('should render value with non-empty initial value', () => {
```

Comment:
This test seems like a duplicate of the second part of the first test.

Reply:
You're right. It's a duplicate since the story has a default value. I think it might still be worth keeping if I use a different value from the default. Or do you still think it would be duplicate?

Reply:
Yeah I think passing in a non-default initial value would a good test to have for sure.